dmcontent/content_loader.py

```diff
@@ -582,6 +623,9 @@ def _questions_path(self, framework_slug, question_set):
     def _message_path(self, framework_slug, message):
         return os.path.join(self._root_path(framework_slug), 'messages', '{}.yml'.format(message))
 
+    def _metadata_path(self, framework_slug, metadata):
```

Comment:
These `_<something>_path` methods are all really similar. Could they be combined and parameterised?

Reply:
They are similar, but I think abstracting this would make some simple methods more confusing than they need to be. The duplication is fine here IMO.